modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -391,23 +400,53 @@ def international_phone_too_long?
         phone.length > 25 if phone
       end
 
```

Comment:
The toxic exposure dates should be covered by the regex in the schema, is it redundant to check them via the `date_is_valid` & 'validate_service_dates' methods? We do this with most other dates, but maybe that is so we can compare the dates to other dates, like with confinements and service dates.